examples/features/load_balancing/server/main.go

```diff
@@ -16,7 +16,8 @@
  *
  */
 
-// Binary server is an example server.
+// Binary server demonstrates how to respond with backend addresses for the
```

Comment:
hmmm...i think server doesn't do anything specific for lb policy. it just returns all the addresses to resolver and its upto client lb policy to use which address to send the request to

Reply:
I think thats what the comment means, that the server responds with the addresses to the client LB policy

Reply:
The server does not return any addresses to the name resolver. Name resolution is a client-side mechanism to discover server backend addresses.

This comment should just say something like
// Binary server spins up multiple server backends to enable client-side load balancing.

Reply:
Done